### DISCUSSION TITLE
[解决]input_plus可以改进一下变速吗

### DISCUSSION THREAD

**Discussion by kli2013** [hooke007/mpv_PlayKit#219] [mpv 通用]
pressaction.lua的变速不会破坏space原来的暂停和播放功能
https://github.com/Natural-Harmonia-Gropius/mpv_config/blob/main/scripts/pressaction.lua

用input_plus的变速绑定space后 ,暂停和播放没了

本来想节约一个脚本,但是还是不行

**Comment by hooke007** [comment]
这两脚本的用途不一样 pressaction.lua 现在最新版叫 inputevent 正如其名把原本只能脚本里使用的 `event` 带到input.conf 里，扩展原本的语法。
而我的脚本旨在扩充更多实用的命令 `commands`

**Reply by kli2013** [reply]
好的

**Reply by hooke007** [reply]
你如果只用那个变速的功能，那确实可以只保留 inputevent 节约一个脚本
